angular.json

```diff
@@ -20,7 +20,7 @@
             "tsConfig": "tsconfig.app.json",
             "aot": false,
             "assets": [
-              "src/favicon.ico",
+              "src/assets/favicon.ico",
```

Comment:
You don't need this line. As the new file is inside `src/assets`.

Reply:
There is no HTML file that manipulate the favicon.